xml/System.Diagnostics/ActivityCreationOptions`1.xml

```diff
@@ -80,8 +82,8 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the name associated with the activity.</summary>
```

Comment:
>Gets the name associated with the activity. [](start = 17, length = 43)

Get the name that will be used as the OperationName of for the Activity that will get create.

Reply:
```suggestion
        <summary>Gets the name to use as OperationName of the activity that will get created.</summary>
```